openrtx/src/ui/default/ui_menu.c

```diff
@@ -83,9 +83,13 @@ static bool DidSelectedMenuItemChange(char* menuName, char* menuValue)
 
     if (strcmp(menuName, priorSelectedMenuName) != 0)
     {
-        strcpy(priorSelectedMenuName, menuName);
+        strncpy(priorSelectedMenuName, menuName, sizeof(priorSelectedMenuName) - 1);
```

Comment:
Please split these changes to a separate commit focused on just the memory issues (just one commit for all of the ui_*.c changes).